src/libnnstreamer-edge/nnstreamer-edge-aitt.h

```diff
@@ -14,6 +14,7 @@
 #define __NNSTREAMER_EDGE_AITT_H__
 
 #include "nnstreamer-edge.h"
+#include "nnstreamer-edge-log.h"
```

Comment:
```#include <aitt_c.h>```, line 26 in this file
The two values below must be redefined and used. This value is defined in <dlog.h> in nnstreamer-edge-log.h.
```
[   41s] In file included from /usr/include/aitt/aitt_c.h:24,
[   41s]                  from /home/abuild/rpmbuild/BUILD/nnstreamer-edge-0.2.5/src/libnnstreamer-edge/nnstreamer-edge-aitt.h:25,
[   41s]                  from /home/abuild/rpmbuild/BUILD/nnstreamer-edge-0.2.5/src/libnnstreamer-edge/nnstreamer-edge-aitt.c:19:
[   41s] /usr/include/aitt/AittTypes.h:71:26: error: expected identifier before numeric constant
[   41s]    71 | #define TIZEN_ERROR_NONE 0
[   41s]       |                          ^
[   41s] In file included from /home/abuild/rpmbuild/BUILD/nnstreamer-edge-0.2.5/src/libnnstreamer-edge/nnstreamer-edge-log.h:26,
[   41s]                  from /home/abuild/rpmbuild/BUILD/nnstreamer-edge-0.2.5/src/libnnstreamer-edge/nnstreamer-edge-aitt.c:23:
[   41s] /usr/include/dlog/dlog.h:72:29: error: 'TIZEN_ERROR_NOT_PERMITTED' undeclared here (not in a function); did you mean 'TIZEN_ERROR_NOT_SUPPORTED'?
[   41s]    72 |  DLOG_ERROR_NOT_PERMITTED = TIZEN_ERROR_NOT_PERMITTED /**< Operation not permitted */
[   41s]       |                             ^~~~~~~~~~~~~~~~~~~~~~~~~
[   41s]       |                             TIZEN_ERROR_NOT_SUPPORTED
[   41s] make[2]: *** [src/CMakeFiles/nnstreamer-edge.dir/build.make:191: src/CMakeFiles/nnstreamer-edge.dir/libnnstreamer-edge/nnstreamer-edge-aitt.c.o] Error 1
```

Reply:
Do you mean that there is a symbol in nnstreamer-edge-aitt.h that refers to aitt_c.h or not?

If yes: include aitt_c.h
If no: don't include aitt_c.h.

Or do you mean that there is a symbol in aitt_c.h that requires including another header before including aitt_c.h?

If yes: fix aitt_c.h or do workaround right before aitt_c.h with proper comments so that we can remove it later. And report to AITT or submit a PR to AITT to fix it.
If no: ?

Reply:
okay, thank you.

Reply:
https://github.com/nnstreamer/nnstreamer-edge/pull/196 
@jaeyun-jung posted a related PR. so I will remove -DTIZEN